NFTEA-Frontend/src/components/CreateAccountFront.vue

```diff
@@ -119,10 +140,42 @@
             </v-card>
           </v-col>
         </v-row>
-      </v-app>
-    </div>
+<!--      </v-app>-->
+<!--    </div>-->
   </v-container>
+</div>
 </template>
+
+
+<style>
+
+  .form {
+    background-color: #D1BE9C;
+    height: 100%;
+  }
```

Comment:
This too could be centered vertically, though it may be a bit tricky as the underlying `v-container` will add its styling after this stylesheet is computed. If you want to, you can try:


```suggestion
  .form {
    background-color: #D1BE9C;
    height: 100%;
    display: flex;
    align-items: center;        /* Vertical center within flexbox   */
    justify-content: center;    /* Horizontal center within flexbox */
  }
```